src/storage-operators/src/s3_oneshot_sink/pgcopy.rs

```diff
@@ -0,0 +1,300 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
this file is just a move of the original `CopyToS3Uploader` and associated tests, the implementation itself was not modified besides renaming to `PgCopyUploader` and having it implement the `CopyToS3Uploader` trait

Reply:
do you mind making this move in a separate commit so its easier to review?

Reply:
ah it should be the 2nd commit in this review: https://github.com/MaterializeInc/materialize/pull/26495/commits/003b742bfb2914a44a9b47f737dbcbaa19c89183

Reply:
@rjobanp ah thanks, will take a look!